Tests/test_SCOP_Scop.py

```diff
@@ -24,16 +24,11 @@ def _compare_cla_lines(self, cla_line_1, cla_line_2):
         """
         fields1 = cla_line_1.rstrip().split("\t")
         fields2 = cla_line_2.rstrip().split("\t")
-        print(fields1)
-        print(fields2)
         # compare the first five fields in a Cla line, which should be exactly
         # the same
-        if fields1[:5] != fields2[:5]:
-            return False
+        self.assertEqual(fields1[:5], fields2[:5])
         # compare the hierarchy key-value pairs, which are unordered
-        if set(fields1[5].split(",")) != set(fields2[5].split(",")):
-            return False
-        return True
+        self.assertCountEqual(fields1[5].split(","), fields2[5].split(","))
```

Comment:
Are the fields always unique?

Reply:
Does it matter?

Reply:
I don't know if it might happen in real SCOP data. Apparently it makes no difference here with the test set, so I guess this change os fine.